plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx

```diff
@@ -93,16 +96,42 @@ export const GroupProfileCard = ({
     ?.filter(r => r.type === RELATION_PARENT_OF)
     ?.map(group => group.target.name);
 
+  const displayName = profile?.displayName ?? name;
+
   if (!group) return <Alert severity="error">User not found</Alert>;
 
   return (
     <InfoCard
-      title={<CardTitle title={name} />}
+      title={<CardTitle title={displayName} />}
       subheader={description}
       variant={variant}
     >
       <Grid container spacing={3}>
-        <Grid item>
+        <Grid item xs={12} sm={2} xl={1}>
+          <Box
+            display="flex"
+            alignItems="flex-start"
+            justifyContent="center"
+            height="100%"
+            width="100%"
+          >
+            <Avatar displayName={displayName} picture={profile?.picture} />
+          </Box>
+        </Grid>
+        <Grid item md={10} xl={11}>
+          {profile?.email && (
+            <Typography variant="subtitle1">
+              <Box display="flex" alignItems="center">
+                <Tooltip title="Email">
```

Comment:
is this for accessibility?

Reply:
I don't know, just moved that code around 🤷‍♂️

Reply:
Aha 😅